qa/automation/CareerPortal.Tests.Scenarios/Artemova/1.2 SendCV.feature

```diff
@@ -0,0 +1,138 @@
+  When user dont enter data into the 'Name' field
+  And moves cursor from the field
+  Then field should turn red
+  And error message 'This field is required' should be shown
+
+Scenario Outline: 1.2.3 User enters invalid data for 'Name' field
+  When user enters invalid <Name> into the 'Name' field
+  And moves cursor from the field
+  Then field should turn red
+  And error message 'Invalid name format' should be shown
+
+  Examples:
+    | Name          | comment                      |
+    | Не английский | not English                  |
+    | 404           | numbers                      |
+    | More than two | more than two words          |
+    | Jane:         | special symbols (,/*-+?^:%;) |
+    | <spaces>      | only spaces                  |
+
+Scenario Outline: 1.2.4 User enters too long data for 'Name' field
+  When user enters too long <Name> into the 'Name' field
+  And moves cursor from the field
+  Then field should turn red
+  And error message 'The maximum number of characters is 70' should be shown
+
+Examples:
+  	| Name          | comment                          |
+  	| too lo(...)ng | over max 70 chars                |
+
+Scenario Outline: 1.2.5 User leaves 'Email' field empty
+  When user dont enter data into the 'Email' field
+  And moves cursor from the field
+  Then field should turn red
+  And error message 'This field is required' should be shown
+
+Scenario Outline: 1.2.6 User enters invalid data for 'Email' field
+  When user enters invalid <Email> into the 'Email' field
+  And moves cursor from the field
+  Then field should turn red
+  And error message 'Invalid Email format' should be shown
+
+Examples:
+  	| Email                  | comment                        |
+  	| Неанглийский@gmail.com | not English                    |
+  	| Mo re@gmail.com        | spaces                         |
+  	| f,g@gmail.com          | special symbols (,/*"'-+?^:%;) |
+  	| fg@gmail               | without domain                 |
+  	| <spaces>               | only spaces                    |
+
+Scenario Outline: 1.2.7 User enters too long data for 'Email' field
+  When user enters too long <Email> into the 'Email' field
+  And moves cursor from the field
+  Then field should turn red
+  And error message 'The maximum number of characters is 70' should be shown
+
+  Examples:
+  	| Email                   | comment           |
+  	| toolo(...)ng@gmail.com  | over max 70 chars |
+
+Scenario Outline: 1.2.8 User enters valid data for 'Message' field
+  When user enter valid <Message> in 'Message' field
+  And clicks 'Send' button
+  Then green tick should be shown in the upper right corner of the field
+
+  Examples:
+    | Message                  | comments                                                           |
+    | 1                        | 1 character                                                        |
+    | In software ... project. | 500 chars, including numbers and special symbols (,.<>"'/*-+?^:%;) |

```

Comment:
"500 chars, including numbers and special symbols (,.<>"'/*-+?^:%;)" should be specified by separate "Message" examples
Comments column is excess